<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDemo65</class>
 <widget class="QDialog" name="dlgDemo65">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bucles usando QThreads</string>
  </property>
  <widget class="QLabel" name="lblBucles">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingresa el Numero de Bucles a Ejecutar</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="txtNumBucles">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000000</number>
   </property>
   <property name="value">
    <number>10000000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEjecutarBucles">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>50</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ejecutar Bucles</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="pbrBucles">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>351</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelarThread">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar Thread</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
